src/OpenRiaServices.Tools/Framework/SharedTypes/SharedCodeService.cs

```diff
@@ -43,7 +43,7 @@ internal SharedCodeService(SharedCodeServiceParameters parameters, ILoggingServi
             ISourceFileProviderFactory[] factories = new ISourceFileProviderFactory[] 
             {
                 new SourceInfoSourceFileProviderFactory(),
-                new PdbSourceFileProviderFactory(symbolSearchPath, loggingService)
```

Comment:
symbolSearchPath seems to not be used anymore and can be deleted

Reply:
Removed